### PR TITLE
Support for Redis Standalone Configuration

### PR DESCRIPTION
**Pull request by aravindanr**

### CODE REVIEW

**Comment by aravindanr** [comment]
> db=redis_standalone

Enables support for a standalone Redis instance. Since _Jedis_ is not threadsafe, _JedisPool_ is used. A corresponding JedisCommands implementation _JedisStandalone_ adapts commands to the Jedis instance provided by JedisPool.

File: redis-persistence/src/main/java/com/netflix/conductor/redis/config/RedisStandaloneConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package com.netflix.conductor.redis.config;
```
Comment (apanicker-nflx) [inline redis-persistence/src/main/java/com/netflix/conductor/redis/config/RedisStandaloneConfiguration.java:1]: Please add license header.

File: redis-persistence/src/main/java/com/netflix/conductor/redis/jedis/JedisStandalone.java [hunk 2]
```diff
@@ -0,0 +1,889 @@
+package com.netflix.conductor.redis.jedis;
```
Comment (apanicker-nflx) [inline redis-persistence/src/main/java/com/netflix/conductor/redis/jedis/JedisStandalone.java:1]: Please add license header.

**Review by apanicker-nflx — COMMENTED** [review]
Also add a conditional to `AnyRedisCondition` so that the DAOs are instantiated.

File: redis-persistence/src/main/java/com/netflix/conductor/redis/jedis/JedisStandalone.java (hunk 2 above)
Comment (aravindanr) [reply redis-persistence/src/main/java/com/netflix/conductor/redis/jedis/JedisStandalone.java:13]: Done

**Pull request merged**